developersociety/commonslibrary
# Fix webpack issues

Before we were using a django webpack bundle loader (not a good idea). This updates to use `webpack-dev-server` and the newer webpack config as used in other projects

The prettier config and formatting means this has ballooned. perhaps add `?w=1` to the url to review without whitespace changes

**To test**
Click around site and make sure interaction and filtering works as expected. I have done a sweep and believe I have covered all the things
